### PR TITLE
feature/#111829345

### PR DESCRIPTION
**Pull request by kkchu791**
https://www.pivotaltracker.com/projects/1516637/stories/111829345

Forgot password part 5 -- remove old forgot_password_uuids

Create custom rake task that removes old forgot_password_uuids from database every day at 12 am
to reduce database size

### CODE REVIEW

File: spec/support/cleaner_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    let!(:user) { FactoryGirl.build(:user) }
+
+    before do
+      load File.expand_path("../../../lib/tasks/cleaner.rake", __FILE__)
+      Rake::Task.define_task(:environment)
+    end
+
+    it "should remove old uuids if 24 hours old" do
+      user.update(forgot_password_uuid:nil)
+      Rake::Task["db:remove_old_uuids"].invoke
```
Comment (kangkyu) [inline spec/support/cleaner_spec.rb]: I guess something should be tested here.. isn't it? Maybe something like `expect(user.forgot_password_uuid).to be_nil` after `invoke` a task.

File: lib/tasks/cleaner.rake [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require 'rake'
+
+namespace :db do
```
Comment (kangkyu) [inline lib/tasks/cleaner.rake]: by naming convention, (this is what they say.. I'm not sure where I heard though) you can name the file and namespace the same. I guess it helps us to find the file later.

**Comment by thomas-jung** [comment]
@kkchu791 this is very cool :+1: . Is using 'db' the convention? I ran it with rake db:remove_old_uuids which is nice. I am wondering if this is the preferred convention or if something like user:remove_old_uuids is acceptable.

**Comment by kkchu791** [comment]
@thomas-jung I'm not sure if naming it db is the correct naming convention. I just chose it because I thought it made sense. I'm sure naming it 'user' would be acceptable too. Thanks for taking a look! :)

**Pull request merged**